pvlib/clearsky.py

```diff
@@ -720,3 +720,103 @@ def rmse(alpha):
         return clear_samples, components, alpha
     else:
         return clear_samples
+
+
+def bird(zenith, airmass_relative, aod380, aod500, precipitable_water,
+         ozone=0.3, pressure=101325., dni_extra=1364., asymmetry=0.85,
+         albedo=0.2):
+    """
+    NREL Bird Simple Clear Sky Broadband Solar Radiation Model
+
+    Based on original implementation by Daryl R. Myers at NREL.
+
```

Comment:
You could add some notes about how zenith and airmass should be calculated if users want to exactly reproduce the entire bird model.